### PR TITLE
Legend refactor

### PR DESCRIPTION
**Pull request by andycochran**
This PR refactors Labs UI components to work with the schema in the `labs-layers-api` branch of the same name. 

**Requires `labs-layers-api` be updated simultaneously.** 

This is a pretty significant change. The best way to understand the changes is probably to run the app and read the docs in the dummy app's `application.hbs`. 

- Legends are now made of stacked SVGs or Font Awesome icons (So robust!) 
- Semantically renames Labs UI components (e.g. `layer-menu-item` -> `layer-group-toggle`) 
- Makes common Sass variables available across Labs projects
- Avoids duplication of Foundation `_settings` file
- Adds documentation for all the components 
- Adds a description of repo to README
- More… _(see dummy app docs)_

### CODE REVIEW

File: tests/dummy/app/models/layer-group.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+// TODO: THIS FILE IF TEMPORARILY OVERRIDING THE MODEL. KILL IT WITH FIRE ONCE EMBER MAPBOX COMPOSER IS UPDATED
```
[the remark this answers is not in the document]
Comment (andycochran) [reply tests/dummy/app/models/layer-group.js]: @chriswhong We made this comment together, I forget how this gets resolved. Remember?

[the remark this answers is not in the document]
Comment (allthesignals) [reply tests/dummy/app/models/layer-group.js]: Probably means we need a new release of composer.

File: addon/components/labs-ui/icon-tooltip.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ import layout from '../../templates/components/labs-ui/icon-tooltip';
 export default class InfoTooltip extends Component {
   layout = layout
 
+  classNames=['icon-tooltip'];
```
[the remark this answers is not in the document]
Comment (andycochran) [reply addon/components/labs-ui/icon-tooltip.js]: Fixed.

**Comment by andycochran** [comment]
@allthesignals I resolved the color contrast ratio warning. It's not a new linter. It's part of Foundation. When you use its built-in color functions, it provides warnings if you're not a11y-compliant. I simply had to change `#ed7d12` to  `#e2750f`, which is hardly perceptible.

**Comment by andycochran** [comment]
@allthesignals I'd be interested to pair on making the dummy app use a static JSON file instead of the API. @chriswhong and I discussed this and thought it was okay since they're so closely tied. Otherwise, if the API changes we have to keep the JSON in sync.

**Pull request merged**